README.md

```diff
@@ -129,6 +148,21 @@ const results = await codeClient.extendAnalysis({
 
 ```
 
+### Run analysis on an existing SCM project and report results to platform
+
+```javascript
+const results = await codeClient.analyzeScmProject({
+  connection: { baseURL, sessionToken, source },
+  analysisOptions: {
+    severity: 1,
+  },
+  reportOptions: {
+    projectId: '<Snyk Project UUID>',
```

Comment:
It would not make sense for this one, only makes sense for the `analyzeFolders` options because that can be done with or without reporting, whereas the SCM flow will always report (i.e. `enabled` is implicitly true).